src/validator/answer.rb

```diff
@@ -30,9 +30,10 @@ module Validator
     class Answer
       ERROR_FILE = File.join(File.dirname(__FILE__), 'errors.yaml').freeze
 
-      def initialize(metalware_config, answer_file)
+      def initialize(metalware_config, answer_file, loader: nil)
```

Comment:
This was added before the `Loader` was in the `config`. We clean that up when I clean the class up as a whole